Add task name to log page (#1918)
taskcluster/taskcluster: changelog/issue-1909.md

> level: minor

Reviewer: ```suggestion
level: patch
```

reference: issue 1909
---

> Taskcluster UI Log pages now have the log name displayed in the header and page title.

Reviewer: ```suggestion
Taskcluster UI log viewer now displays the log name in the app bar.
```

Author: I committed the other suggestion via GitHub without adding to batch (oops), and I'll have to update this separately. Sorry for making too many commits on this PR.

Reviewer: No worries. Just let me know when this is ready for review.

Author: Thanks @helfi92 it's ready for review.
